### PR TITLE
Added Wishlist page

### PR DESCRIPTION
**Pull request by surbhi84**
not responsive yet

.

### CODE REVIEW

File: pages/cart/index.html [hunk 1]
```diff
@@ -64,22 +64,9 @@ <h2 class="mg-xs">Hallo!Decor</h2>
     </header>
     <!-- showcase for quicklinks by categories -->
     <div class="parting flex-row">
-      <a href="/pages/productList/" class="text-dec-none text-link"
-        >Art & Decor</a
+      <a href="/pages/productList/" class="text-dec-none text-link med-text"
```
Comment (ckahub) [inline pages/cart/index.html:67]: avoid using extra space

File: pages/wishlist/index.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+              <a
+                href="https://github.com/surbhi84"
+                target="blank"
+                class="footer-icon"
+                ><i class="fab fa-github"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://twitter.com/Surbhikukreti89"
```
Comment (ckahub) [inline pages/wishlist/index.html:136]: why so much space used

**Review by ckahub — COMMENTED** [review]
good to merge

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: pages/wishlist/index.html (hunk 2 above)
Comment (surbhi84) [reply pages/wishlist/index.html:136]: this space is added by the linting library it increases the readability of the code.

**Pull request merged**